### PR TITLE
Modify the TPC-H as suggested and general test files of duckdb, pandas, polars, and cuDF

### PR DESCRIPTION
**Pull request by mia1460**

### CODE REVIEW

File: tpch/pandas_queries/queries.py [hunk 1]
```diff
@@ -513,6 +513,7 @@ def q08(root: str, storage_options: Dict):
         (orders["O_ORDERDATE"] >= pd.Timestamp("1995-01-01"))
         & (orders["O_ORDERDATE"] < pd.Timestamp("1997-01-01"))
     ]
+    orders_filtered = orders_filtered.copy()
```
Comment (luweizheng) [inline tpch/pandas_queries/queries.py:516]: Do we really need a copy()?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
 <h3 align="center">
-    <p>DF-Eval: a multi-level, multi-dimensional evaluation suite for DataFrame systems.</p>
+    <p>DF-Eval: a multi-level, multi-dimensional evaluation suite for DataFrame systems.
+    </p>
```
Comment (luweizheng) [inline README.md:3]: no need to modify code here

File: tpch/pandas_queries/queries.py [hunk 3]
```diff
@@ -694,6 +695,12 @@ def g2(x):
     total = total.sort_values("L_SHIPMODE").rename(
         columns={"g1": "HIGH_LINE_COUNT", "g2": "LOW_LINE_COUNT"}
     )
+
+    # Round the result to one decimal place
```
Comment (luweizheng) [inline tpch/pandas_queries/queries.py:699]: I think we do not need to cast type to float.

File: tpch/polars_queries/queries.py [hunk 4]
```diff
@@ -7,6 +7,7 @@
 from typing import Dict
 
 import polars as pl
+import sys
```
Comment (luweizheng) [inline tpch/polars_queries/queries.py:10]: no need to import sys here

File: tpch/polars_queries/queries.py [hunk 5]
```diff
@@ -581,12 +592,12 @@ def q14(root: str, storage_options: Dict):
     line_item_ds = load_lineitem_lazy(root, storage_options)
     part_ds = load_part_lazy(root, storage_options)
 
-    var_1 = datetime(1995, 9, 1)
-    var_2 = datetime(1995, 10, 1)
+    startDate = datetime(1994, 3, 1)
```
Comment (luweizheng) [inline tpch/polars_queries/queries.py:595]: variable name should be start_date. Keep it consistent in other places.

File: tpch/polars_queries/queries.py [hunk 6]
```diff
@@ -633,44 +643,51 @@ def q15(root: str, storage_options: Dict):
     return q_final
 
 
+# 大改 https://github.com/pola-rs/tpch/blob/main/queries/polars/q16.py 有问题
```
Comment (luweizheng) [inline tpch/polars_queries/queries.py:646]: Please do not have any chinese characters. Check these problems in all other places.

File: tpch/duckdb_queries/queries.py [hunk 7]
```diff
@@ -137,9 +137,9 @@ def q02(root: str):
     part = load_part(root)
     partsupp = load_partsupp(root)
 
-    SIZE = 15
-    TYPE = "BRASS"
-    REGION = "EUROPE"
+    size = 15
+    type = "BRASS"
+    Region = "EUROPE" # Different from the region table, the same below.
```
Comment (luweizheng) [inline tpch/duckdb_queries/queries.py:142]: Region -> region

File: tpch/pandas_queries/queries.py (hunk 1 above)
Comment (mia1460) [reply tpch/pandas_queries/queries.py:516]: Yes, it is indeed possible not to add it, but the compiler recommends it. For the sake of time testing, I will delete it.

**Pull request merged**